### ISSUE TITLE
[jshow] Accessibility issue with search Icon & Button 

### ISSUE THREAD

**Issue by arunsasi** [joomla/joomla-websites#157]
http://joomla-showcase.octopoos-dev.com/

Failed color contrast Orange icon & Button  background-color: #ff8100;
![failed](https://cloud.githubusercontent.com/assets/270789/11013716/7096a4ea-853e-11e5-9b09-7e2346dce91b.png)

**Comment by oliviernolbert** [comment]
Fixed, "Go" action button is now with blue background.

Thanks

**Comment by MATsxm** [comment]
Closed - Reopen if needed
